### PR TITLE
WEB-2068 showLoadingOverlay / hideLoadingOverlay

### LINKED ISSUE #145: Show/Hide Loading Overlay
Due this [spec](https://confluence.tid.es/display/CTO/%5BAPPS%5D+Tech+Spec%3A+Tela+de+Loading), we need to provide the following 2 bridge methods:

## Show Loading Overlay

This method allows showing a fullscreen loading overlay following [this figma specification](https://www.figma.com/design/Via0lhrPKYB6DOQMbWcshO/Loading-animation?node-id=1-3&node-type=canvas&t=8rfucRUlSUQEaEy9-0)

All method parameters are optional, if method is called while a previous loading screen is being shown, it would return an error.

### Js method
```javascript
showLoadingOverlay: ({
    inAnimation?: boolean;
    outAnimation?: boolean;
    minimumLoopDurationMs?: number;
    stopAnimationCycle?: boolean;
    backgroundAnimation?: boolean;
    descriptions?: Array<string>;
    descriptionDurationMs?: number;
    timeoutMs?: number; // After this timeout loading screen would be hidden automatically
    closeOnBackButtonPressAfterFinish?: boolean; // (Only Android) If true, after loading screen has been hidden, if user presses android back button, webview window will close.
}) => Promise<void>
```

Default values (Just for documentation purposes, determined in native apps):

```
inAnimation --> false
outAnimation --> false
minimumLoopDurationMs --> 0
stopAnimationCycle --> true
backgroundAnimation --> false
descriptions --> []
descriptionDurationMs --> 3000
timeoutMs --> 20000
closeOnBackButtonPressAfterFinish --> true
```

Message type:

**SHOW_LOADING_OVERLAY**

Error response codes
- 503 --> Loading screen already showing.

## Hide Loading Overlay

This method allows hiding the fullscreen loading overlay previosuly mentioned.

In case it is called when the loading screen is not currently shown, it would have no effect.

### Js method
```javascript
hideLoadingOverlay = ( ) => Promise<void>
```

Message type:

**HIDE_LOADING_OVERLAY**

Error response codes
- No errors

### PR DESCRIPTION
**Pull request by atabel**
closes #145

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1257,6 +1257,70 @@ Writes the given text to the clipboard
 writeTextToClipboard: (text: string) => Promise<void>;
 ```
 
+### showLoadingOverlay / hideLoadingOverlay
```
Comment (atabel) [inline README.md:1260]: @dpastor, if you have a video with a demo I can add it to the docs

File: README.md [hunk 2]
```diff
@@ -1257,6 +1257,70 @@ Writes the given text to the clipboard
...
+
+<kbd>App version >=24.10</kbd>
```
Comment (atabel) [inline README.md:1262]: is this min app version ok, @dpastor ?

Comment (dpastor) [reply README.md:1262]: Yes, thanks :)

File: README.md (hunk 1 above)
Comment (dpastor) [reply README.md:1260]: We could add this one (It's not yet verified, so maybe something changes until release):

https://github.com/user-attachments/assets/06613a96-28bd-4b26-bd84-ba5e4add4ae9

**Review by marcoskolodny — APPROVED** [review]

**Review by dpastor — APPROVED** [review]
Thanks! :)

**Pull request merged**